tests/test_suites/ShortSystemTests/test_ec_prioritize_data_parts.sh

```diff
@@ -0,0 +1,61 @@
+timeout_set 1 minute
+
+CHUNKSERVERS=4 \
+	DISK_PER_CHUNKSERVER=1 \
+	MOUNT_EXTRA_CONFIG="sfscachemode=NEVER" \
+	MASTER_CUSTOM_GOALS="10 ec_3_1: \$ec(3,1)" \
```

Comment:
Consider also testing this algorithm with different label on chunkservers and the `USE_LINEAR_ASSIGNMENT_OPTIMIZER` option enable on master.